src/components/shmoke.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+<div id='s' class="fl ma2 tc b--solid bg-green shadow-5 db pa3"> 
+        <h2>🌲 SHMOKE up FELLA 🌲 </h2>
+        <img class="snoop" @click="shmoke_weed"  :src = getimageUrl() />
+</div>     
+
+</template>
+
+
+<script>
+import eventBus from '../eventBus';
+export default {
+    data(){
+        return {
+            shmoking: [
+                {
+            shmoked:true, 
+            name:"shmokie",
+            run: () => {
+                eventBus.$emit('achievement-unlocked' , 'We only smoke quads')
+                eventBus.$emit('big-kush-energy')
+                this.$modal.show('dialog', {
+              title: `<h1> SHMOKING GANJA BIG BOI , ENJOY A 20 🍆 BOOST </h1>`,
+              buttons:[{
+                  title: "puff puff 🅱A🅱Y"
+              }]
+                }) 
+            }
+                }
+            ]
+        }
+    },
+    computed:{
+        shmoke(){
+           return this.shmoking
+        }
+    },
+    methods:{
+        getsoundUrl(){
+            return require('../assets/items/shmoke.mp3');
+        },
+        getimageUrl() {
+            return require('../assets/items/tenor.gif')
+        },
+
+        shmoke_weed() {
+            this.shmoke[0].run()
+        }
+    }
+
+    
+}
+</script>
+
+<style scoped>
+    h2{
+        font-size: 20px;
+    }
+    #s{
+        height: 450px;
+        width: 300px;
+        transition: background-image  1s  ease-in-out;
```

Comment:
this is actually css3 so it might not do the animation,  but it still does the transition